knowledge-base/chart-display-empty.md

```diff
@@ -0,0 +1,107 @@
+---
+title: How to Show Empty Chart Instead of the Default No Data Template
+description: Learn how to show an empty Chart component when there is no data, instead of displaying the default No Data template.
+type: how-to
+page_title: How to Show Empty Chart Instead of the Default No Data Template
+slug: chart-kb-display-empty-chart
+tags: charts, blazor, no data template, empty chart
+res_type: kb
+ticketid: 1675313
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+As of version 7.1.0 of Telerik UI for Blazor, the [No Data Template](slug://chart-no-data-template) was introduced for Charts in Blazor. In some scenarios, displaying an empty Chart, rather than the No Data template, is preferred when there is no data. This knowledge base article also answers the following questions:
+
+- How can I hide the No Data Template in Blazor Charts?
+- Is it possible to display an empty Chart in Blazor when there's no data?
+- Can I override the No Data Template in Blazor Charts?
```

Comment:
It seems unnecessary to have "This knowledge base article also answers the following questions:" section, given the description.